force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -126,16 +156,19 @@ public with sharing class ClassesObjectsMethods {
         List<String> scientistNames = new List<String>{'Tim Berners-Lee', 'Alan Turing', 'Grace Hopper', 'Donald Knuth', 'Guido van Rossum', 'Ken Thompson', 'Stephen Hawking'};
 
         // The variable to store the concatenated string
+         String scientists = '';
 
         // Loop through the list of scientists' names
 
-            // If the name doesn't contain the letter 'G', skip this iteration
+        For (String names : scientistNames) {
+            if (names.toLowerCase().contains('g')){
+                scientists += names+',';                
 
-            // Add the name to the result string, followed by a comma
+            }
 
-        // Remove the last comma from the result string
-
-        return null; // Replace null with the variable you used to store the result
+        }
+        
+        return scientists.removeEnd(','); 
```

Comment:
You can store the names to a List, then you use String.join to join them with a comma. You won't need to remove the trailing comma.

Reply:
Thank you. I was thinking there was a simpler way. Thank you.